modules/gitops-configuring-tls-for-redis-with-autotls-enabled.adoc

```diff
@@ -0,0 +1,114 @@
+// Module is included in the following assemblies:
+//
+// * /cicd/gitops/configuring-secure-communication-with-redis.adoc
+
+[id="gitops-configuring-tls-for-redis-with-autotls-enabled_{context}"]
+= Configuring TLS for Redis with autotls enabled
+
+[NOTE]
+====
+By default, the `autotls` setting is disabled.
+====
+
```

Comment:
@ekristova You need to add an introductory sentence or two for this procedure module file. See here for details on the different parts of a procedure module: https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs